Makefile

```diff
@@ -31,7 +35,12 @@ build-translation:
 	$(NPM) run translate --prefix axolotl-web
 
 run: build
-	$(GO) run .
+	ifdef GO_VERSION
```

Comment:
What about assuming that go, cargo, git and other required tooling already is installed? Without it, we can keep the Makefile slimmer and I am assuming that the person using the command will get a big error message, along the lines of "go not found" which in itself is verbose enough.